crd/fleet/fleet-crds.yaml

```diff
@@ -7765,9 +9488,14 @@ spec:
                   The `Switchover` object must be created in the same namespace as
                   the DBCluster that it references. This field is required for Switchover.
                 type: string
+              newPrimary:
+                description: NewPrimary is the standby instance to switch with the
+                  current primary.
+                type: string
               newPrimaryInstance:
-                description: NewPrimaryInstance is the standby instance to switch
-                  with the current primary.
+                description: 'NewPrimaryInstance is the standby instance to switch
```

Comment:
Quotation marks from the beginning and end of the sentence.

Reply:
do I need to update this to double quotes?

Reply:
Yes. Thank you!